Makefile

```diff
@@ -0,0 +1,23 @@
+CHECK = \033[32m✔\033[39m
+PROJECT_NAME=rabbitmq-worker
+IMAGE_NAME=tsilvers/$(PROJECT_NAME)
+GOPATH=$(shell pwd)/vendor:$(shell pwd)
+GOBIN=$(shell pwd)/bin
+
+build: get binary docker
+
+get:
+	@echo -n "Getting dependencies...                          "
```

Comment:
Just copying the style I saw in https://github.com/LinioIT/shop-front-queue-listener/blob/master/Makefile. Added my own twist to display the trailing checkmark after the command completes.